api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/tenant_dashboards_controller.rb

```diff
@@ -0,0 +1,31 @@
+module MnoEnterprise::Concerns::Controllers::Jpi::V1::Impac::TenantDashboardsController
+  extend ActiveSupport::Concern
+
+  #==================================================================
+  # Included methods
+  #==================================================================
+  # 'included do' causes the included code to be evaluated in the
+  # context where it is included rather than being executed in the module's context
+  included do
+    respond_to :json
+  end
+
+  DASHBOARD_DEPENDENCIES = [:widgets, :'widgets.kpis', :kpis, :'kpis.alerts', :'kpis.alerts.recipients']
+
+  #==================================================================
+  # Instance methods
+  #==================================================================
+  # GET /mnoe/jpi/v1/impac/tenant_dashboards
+  def index
+    dashboards
+    render template: 'mno_enterprise/jpi/v1/impac/dashboards/index'
+  end
+
+  private
+
+  def dashboards
+    @dashboards ||= MnoEnterprise::Dashboard
+      .includes(*DASHBOARD_DEPENDENCIES)
+      .find(owner_type: 'Mnoe::Tenant')
```

Comment:
When is the dashboard created with `owner_type = 'Mnoe::Tenant'`? Is it directly in mnohub?

Reply:
As you now know, currently it is not being created, as I got this particular element of the project confused with the SPA and thought that we were also provisioning a dashboard for the tenant.